lib/gh-koenig/.gitignore

```diff
@@ -0,0 +1,55 @@
+b-cov
```

Comment:
Do we need the additional `.gitignore` file? All of these ignores should already be in the root `.gitignore`